xil_carma_cloud/cdasim_config/carma/inlanecruising_config/parameters.yaml

```diff
@@ -4,7 +4,7 @@ curve_resample_step_size: 1.0 # Curve re-sampling step size in m
 default_downsample_ratio: 18 # Amount to downsample input lanelet centerline data. Value corresponds to saving each nth point.
 turn_downsample_ratio: 20 # Amount to downsample input lanelet centerline data on turns. Value corresponds to saving each nth point.
 #minimum_speed: 2.2352 # Minimum allowable speed in m/s
-minimum_speed: 5.0 # Minimum allowable speed in m/s
+minimum_speed: 2.2352 # Minimum allowable speed in m/s
```

Comment:
Why are we making this change.

Reply:
Please see the description, it helps make the vehicle turn. It is what I observed in the SIM PC1